resources/views/layouts/partials/header/account.blade.php

```diff
@@ -9,6 +9,7 @@
                 @if(App::providerIsLoaded('Rapidez\Account\AccountServiceProvider'))
                     <a class="block hover:bg-secondary px-3 py-2" href="/account">@lang('Account')</a>
                     <a class="block hover:bg-secondary px-3 py-2" href="/account/orders">@lang('Orders')</a>
+                    <a class="block hover:bg-secondary px-3 py-2" href="/account/wishlist">@lang('Wishlist')</a>
```

Comment:
This page isn't included within the account page right? So if we remove the package the link is still present.